## Hi there ðŸ‘‹

I am a Research Fellow at the University of Oxford studying the evolution of microbes, particularly pathogenic viruses and bacteria. I have a diverse background in genomics, population genetics, phylogenetics, and community ecology. My current research involves developing novel computational methodologies and software tools and applying them to large-scale genomic datasets.

<details>
<summary>Ancestral recombination graphs</summary>
<br>
  <ul>
    <li>â€‹Jeffery B, et al. 2025. Population-scale Ancestral Recombination Graphs with tskit 1.0. arXiv. https://arxiv.org/abs/2602.09649</li>
    <li>Deng Y, <b>Zhan SH</b>*, Zhang Y*, Zhang C, Chen B. 2025. Tree Thinking in the Genomic Era: Unifying Models Across Cells, Populations, and Species. *Equal contribution. arXiv. https://arxiv.org/abs/2512.05499</li>
    <li><b>Zhan SH</b>,â€‹ Wong Y*,â€‹ Ignatieva A*â€‹, Eaton Kâ€‹, Guthrie Iâ€‹, Jeffery Bâ€‹, Palmer DSâ€‹, Murall CLâ€‹, Otto SP, Kelleher J. 2025. A Pandemic-Scale Ancestral Recombination Graphs for SARS-CoV-2. *Equal contribution. bioRxiv. https://www.biorxiv.org/content/10.1101/2023.06.08.544212v3</li>
    <li><b>Zhan SH</b>, Ignatieva A*â€‹, Wong Y*, Eaton Kâ€‹, Jeffery Bâ€‹, Palmer DSâ€‹, Murall CLâ€‹, Otto SP, Kelleher J. 2023. Towards Pandemic-Scale Ancestral Recombination Graphs of SARS-CoV-2. *Equal contribution. bioRxiv. https://www.biorxiv.org/content/10.1101/2023.06.08.544212v1</li>
  </ul>
</details>

<details>
<summary>Chromosomal evolution</summary>
<br>
  <ul>
    <li>Li Z*, Kinosian SP*, <b>Zhan SH</b>, Barker MS. Ancient polyploidy and low rate of chromosome loss explain the high chromosome numbers of homosporous ferns. *Equal contribution. In revision. https://www.biorxiv.org/content/10.1101/2024.09.23.614530v1</li>
    <li><b>Zhan SH</b>, Otto SP, Barker MS. Broad variation in rates of polyploidy and dysploidy across flowering plants is correlated with lineage diversification. In revision. https://www.biorxiv.org/content/10.1101/2021.03.30.436382v1</li>
    <li><b>Zhan SH</b>, Drori M, Goldberg EE, Otto SP, Mayrose I. 2016. Phylogenetic evidence for cladogenetic polyploidization in land plants. <b>Am. J. Bot.</b> https://bsapubs.onlinelibrary.wiley.com/doi/full/10.3732/ajb.1600108</li>
    <li><b>Zhan SH</b>, Glick L, Tsigenopoulos CS, Otto SP, Mayrose I. 2014. Comparative analysis reveals that polyploidy does not decelerate diversification in fish. <b>J. Evol. Biol.</b> https://academic.oup.com/jeb/article-abstract/27/2/391/7318530</li>
    <li>Mayrose I, <b>Zhan SH</b>, Rothfels CJ, Magnuson-Ford K, Barker MS, Rieseberg LH, Otto SP. 2011. Recently formed polyploid plants diversify at lower rates. <b>Science</b> https://www.science.org/doi/abs/10.1126/science.1207205</li>
  </ul>
</details>

<details>
<summary>Community ecology</summary>
<br>
  <ul>
    <li><b>Zhan SH</b>, Huang YT, Liao CP, Liou CY, Liu SL. 2025. Sedimentation-driven environmental filtering drives community assembly of cryptic coralline algae. In revision.</li>
    <li><b>Zhan SH</b>, et al. 2022. Geographic distance, sedimentation, and substrate shape cryptic crustose coralline algal assemblages in the worldâ€™s largest subtropical intertidal algal reef. <b>Mol. Ecol.</b> <i>On the Cover.</i> https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.16455</li>
    <li><b>Zhan SH</b>, Hsieh TY, Yeh LW, Kuo TC, Suda S, Liu SL. 2020. Hidden introductions of freshwater red algae via the aquarium trade exposed by DNA barcodes. <b>Environ. DNA.</b> https://doi.org/10.1002/edn3.139</li>
    <li><b>Zhan SH</b>, Shih CC, Liu SL. 2020. Reappraising plastid markers of the red algae for phylogenetic community ecology in the genomic era. <b>Ecol. Evol.</b> https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.5984</li>
    <li>Hsieh CJ*, <b>Zhan SH*</b>, Liao CP*, et al. 2018. The effects of contemporary selection and dispersal limitation on the community assembly of acidophilic microalgae. <b>J. Phycol.</b> https://onlinelibrary.wiley.com/doi/abs/10.1111/jpy.12771</li>
    <li>Hsieh CJ, <b>Zhan SH</b>, Lin Y, Tang SL, Liu SL. 2015. Analysis of rbcL sequences reveals the global biodiversity, community structure, and biogeographical pattern of thermoacidophilic red algae (Cyanidiales). <b>J. Phycol.</b> https://onlinelibrary.wiley.com/doi/abs/10.1111/jpy.12310</li>
  </ul>
</details>

<!--
**szhan/szhan** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on ...
- ðŸŒ± Iâ€™m currently learning ...
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
